### ISSUE TITLE
Setup a registration portal and a mechanism for sponsors to pay us

### ISSUE THREAD

**Issue by moorepants** [mechmotum/bmd2023#7]
https://intranet.tudelft.nl/-/aanmelder-nl?p_l_back_url=%2Fsearch%3Fq%3Daanmelder

https://www.aanmelder.nl/tutorial/wiki/571675/bibliotheek

https://knowledge.aanmelder.nl/nl/knowledge

**Comment by moorepants** [comment]
Some updates:

- You can log into aanmelder using a TU Delft netid but you can't connect it to a financial unit or account at TU Delft without someone at TU Delft called a "cco" or an "owner". We have asked around extensively, but haven't been able to find who this is.
- TUD Events services told us that aanmelder is the only approved service like this and we wanted to use something like stripe.com (what orvium offers) then it is a months long bureaucracy path to get it approved.
- There are many aanmelder registration websites for TU Delft events: see this search https://www.google.com/search?q=tu+delft+site%3Aaanmelder.nl, but hardly any list people's contact info on those sites. If we could contact someone connected to one of the recent sites, they could help us figure out who to talk to.
- Aanmelder.nl did not seem willing to give us a name of anyone at TU Delft that is the "cco" or "owner".
- We asked around extensively at TUD and eventually were told to speak to Arno Elert who is an "information specialist". He seems to have the power at TUD to enable a registration site for us but apparently this takes a long time (month) to setup also.
- Arno connected us to Rianna Kaasschieter who can handle financial things. She led us to this intranet link https://intranet.tudelft.nl/-/creating-sales-invoices-for-courses-conferences-and-symposia?redirect=%2Fdienst-aanvragen  which describes how we can initiate invoices (for the sponsor payments). This is apparently the appropriate way to do this, instead of letting sponsors also pay through aanmelder.nl.

**Comment by moorepants** [comment]
We finally were able to set this up.

Once you log into aanmelder.nl with you TU Delft authentication it was (or eventually became) possible to set the financial connection to TU Delft 3mE. I then provide the baan (dim) code that I want the money to go to in a field beside that selection. We selected the option to let aanmelder manage the money and then send a lump sum after the conference. This should end up in the baan code at 3me. We also selected to include the VAT on the display to the registrants and set it to 21% which Edwin found out is the standard number for conferences.
